src/main/kotlin/app/cash/licensee/outputModel.kt

```diff
@@ -25,6 +25,7 @@ data class ArtifactDetail(
   // TODO do we need to include extension and classifier?
   val spdxLicenses: Set<SpdxLicense> = emptySet(),
   val unknownLicenses: Set<UnknownLicense> = emptySet(),
+  val scm: ArtifactScm? = null,
```

Comment:
Any reason to nest it in an object rather than a direct property?

Reply:
Just left as a toe-hold in case you want to add other scm properties in the future (`connection`, etc). Can make it just a simple property if you want, was just thinking an object was less brittle to any future format breaks